src/Common/tests/DefaultOptionsProviderTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// ***********************************************************************
```

Comment:
We could leave it in the Silverlight build, provided there is a way for TC to launch a Silverlight test. My thought is that that would require creating a special command-line runner, however. We have an issue for that too. :-)